worlds/ff6wc/test/test_verify_flags.py

```diff
@@ -8,8 +8,8 @@ def test_ok_flags(self) -> None:
         verify_flagstring([])
 
     def test_new_flags(self) -> None:
-        """ These flags are from a newer version of Worlds Collide that isn't supported yet. """
-        self.assertRaises(ValueError, verify_flagstring, ["-i", "x", "-chrm", "0", "0"])
+        """ some new flags from Worlds Collide 1.4.2 """
+        verify_flagstring(["-i", "x", "-chrm", "0", "0"])
 
     def test_bad_flags(self) -> None:
         self.assertRaises(ValueError, verify_flagstring, ["-i", "x", "-bkbkb00"])
```

Comment:
Are there any flags that are valid in 1.2 and not valid in 1.4?

It might be good to have a test to make sure the 1.2 flag makes an error.

Reply:
Zozo Clock?

Reply:
> Zozo Clock?

-rc is deprecated, but is still accepted (then ignored) by the base FF6WC processing, so no need to add any processing there. 
This is the same case with a few of the options that were deprecated from 1.2.2 -> 1.4.2, thus no need to add any new values into test_bad_flags